index.html

```diff
@@ -60,131 +79,146 @@ <h3 class="job-title">
                 <h2 class="recentWorks">
```

Comment:
- [ ] What the previous reviewer meant here was the following:- 
  -  First, without hovering the mouse on the card, the `see button` should not be there. 
  -  Then, when hovering the mouse on card, the description and dark opacity should disappear and `see button` shows up. you can achieve this by using `:hover` pseudoclass, and not with the click.
Right now, the hover effect is not working on these cards. I had to click, and even on clicking, the desired effect is not there.
 your design now.

https://user-images.githubusercontent.com/96683843/229208198-c4b7b2f1-c3ff-4151-b7af-44e044e36299.mov